### DISCUSSION TITLE
Concatenate the values of the two fields as a new field output

### DISCUSSION THREAD

**Discussion by kartikzheng** [fluent/fluent-bit#5735] [Q&A] [answered]
Can I concatenate the values of the two fields as a new field output
Such as
A: "hello"
B: “world”

output:
C: "hello world"

thanks.

**Comment by agup006** [comment, marked as answer]
I think the easiest for this would be to do it in a lua filter. @tarruda might have an example lurking around

**Reply by tarruda** [reply]
Here's a simple Lua filter that would create a new field "c" that has fields "a" and "b" concatenated:

```lua
function cb_filter(tag, ts, record)
  record.c = record.a .. record.b
  return 1, ts, record
end
```

**Reply by kartikzheng** [reply]
I see. Thank you very much

**Reply by tarruda** [reply]
Not sure you have checked already, but you can easily test lua filter snippets using our [playground web app](https://calyptia.github.io/sandbox/)

**Reply by tarruda** [reply]
BTW it is still experimental, but we are working to improve it. For example, right now there's no feedback about syntax errors on the UI, but you can see those on the devtools console (F12)

**Reply by kartikzheng** [reply]
OK，Thanks for sharing.
